test/Interop/Cxx/foreign-reference/Inputs/cxx-functions-and-methods-returning-frt.h

```diff
@@ -280,3 +281,51 @@ template <typename T>
 FRTStruct
     *_Nonnull global_templated_function_returning_FRT_create_with_attr_returns_unretained(
         T a) __attribute__((swift_attr("returns_unretained")));
+
+_Pragma("clang diagnostic push")
+    _Pragma("clang diagnostic ignored \"-Wnullability-extension\"")
+        _Pragma("clang assume_nonnull begin")
+
+            template <typename T>
+            T global_function_returning_templated_retrun_frt(T);
```

Comment:
Minor: this seems to be mis-formatted, could you please run clang-format on this code?